### PR TITLE
{chem}[intel/2020b] molecularGSM v20190826

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node354.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/1bfaa6ae448a138db0e7dd59b33463f7 for a full test report.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/m/molecularGSM/molecularGSM-20190826-intel-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    # '-DGSM_ENABLE_QCHEM_SF=1',
+    # '-DGSM_ENABLE_ORCA=1',
+    '-DGSM_ENABLE_GAUSSIAN=1',
+    # '-DGSM_ENABLE_MOLPRO=1',
+    # '-DGSM_ENABLE_ASE=1',
+]
+
+install_cmd = 'mkdir -p %(installdir)s/bin && cp -a GSM/*.exe %(installdir)s/bin/'
+
+# _bins = ['ase', 'gaussian', 'molpro', 'mopac', 'orca', 'qchem', 'qchem_sf']
```
Comment (verdurin) [inline easybuild/easyconfigs/m/molecularGSM/molecularGSM-20190826-intel-2020b.eb:55]: @smoors could you add some explanation for the commented out test? I assume this is so people can customise which parts are enabled for the build?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/m/molecularGSM/molecularGSM-20190826-intel-2020b.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/m/molecularGSM/molecularGSM-20190826-intel-2020b.eb]: it was left-over from some local testing, I've uncommented it now.

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/0ec4e34d3c04f657412503666fe14c3c for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @smoors!

**Pull request merged**